File: android/app/src/main/java/com/ismin/csproject/PoI.kt
package com.ismin.csproject

import java.io.Serializable

data class PoI(
    var name: String,
    var place: String,
    var latitude: Double,
    var longitude: Double,
    var level: String,
    var type: String,
    var favorite: Boolean
    ): Serializable

File: api/src/PoI.ts
export interface PoI {
    name: string;
    place: string;
    latitude: number;
    longitude: number;
    level: string;
    type: string;
    favorite: string;
}

export interface DetailedPoI {
    name: string;
    place: string;
    latitude: number;
    longitude: number;
    commentary: string;
    level: string;
    type: string;
    pictureLink: string;
    favorite: string;
}

export interface ExternalPoI {
    Name: string;
    Lieu: string;
    GPS_Coord: string;
    Badge: string;
    Commentaires: string;
    Niveau: string;
    Type: string;
    gx_media_links: string;
}